### PR TITLE
Restructure volume C and B

### PR DESCRIPTION
**Pull request by jricher**
Add page headers to volume C and B, add links to C and B pages to sidebar.

### CODE REVIEW

File: _includes/sidenav.html [hunk 1]
```diff
@@ -15,7 +15,20 @@
 
 		<li class="dropdown-menu"><a href="{{ site.baseurl }}/63A/">SP 800-63A</a></li>
 		<li class="dropdown-menu"><a href="{{ site.baseurl }}/63B/">SP 800-63B</a></li>
+      <a href="{{ site.baseurl }}/63B/Terminology">Terminology</a>
+      <a href="{{ site.baseurl }}/63B/AAL">Authenticator Assurance Levels</a>
+      <a href="{{ site.baseurl }}/63B/Authenticators">Authenticators</a>
+      <a href="{{ site.baseurl }}/63B/Lifecycle">Lifecycle Management</a>
+      <a href="{{ site.baseurl }}/63B/Session">Session Management</a>
+      <a href="{{ site.baseurl }}/63B/SampleAuthenticators">Sample Authenticators</a>
```
Comment (jimfenton) [inline _includes/sidenav.html:24]: Do these new links need to be enclosed inside a
    ```<li class="dropdown-item">  </li>```
like the 63C sections are?

File: _includes/sidenav.html [hunk 2]
```diff
@@ -15,7 +15,20 @@
...
 		<li class="dropdown-menu"><a href="{{ site.baseurl }}/63C/">SP 800-63C</a></li>
+    <li class="dowpdown-item">
```
Comment (jimfenton) [inline _includes/sidenav.html]: dowpdown -> dropdown

**Review by jimfenton — CHANGES_REQUESTED** [review]
close...

**Comment by jricher** [comment]
Good catch, fixed both of those. Note that the sidebar doesn't have the expand/collapse functionality yet so it goes off screen, but that needs to be added regardless.

**Review by jimfenton — APPROVED** [review]

**Pull request merged**